app/assets/stylesheets/govuk_publishing_components/components/_show-password.scss

```diff
@@ -49,6 +49,7 @@
 
   &:hover {
     color: $govuk-link-hover-colour;
+    @include govuk-link-hover-decoration;
```

Comment:
Wondering if all text within buttons should follow the same convention - eg

![image](https://user-images.githubusercontent.com/71266765/120324267-56d10800-c2de-11eb-83ec-7395453224f2.png)